source/sdk/Memory.ooc

```diff
@@ -2,17 +2,8 @@ include string // Memory routines in C are actually in string.h
 version(windows) { include malloc }
 else { include alloca }
 
-// GC_MALLOC zeroes the memory, so in the non-gc version, we prefer to use calloc
-// to the expense of some performance. Don't use malloc instead - existing ooc code
-// is written assuming that gc_malloc zeroes the memory.
-gc_malloc: func (size: SizeT) -> Pointer { gc_calloc(1, size) }
```

Comment:
Why do we do this ? Previously it was possible to redefine `gc_calloc` and `gc_free` like
```ooc
gc_calloc: func (nmemb, size: SizeT) -> Pointer {
    result := calloc(nmemb, size)
    "%i bytes allocated at address %i" printfln(nmemb, result)
    result
}

gc_free: func (address: Pointer) {
    "free memory at address %i" printfln(address)
    free(address)
}
```
or add another form of tracking / logging memory allocations. I did that at least once.
I think it won't be possible after this change.